File: core/src/bara/game/systems/ParallaxSystem.java
package bara.game.systems;

import com.badlogic.ashley.core.Entity;
import com.badlogic.ashley.core.Family;
import com.badlogic.ashley.systems.IteratingSystem;
import bara.game.components.ParallaxComponent;

public class ParallaxSystem extends IteratingSystem {

    public ParallaxSystem() {
        super(Family.all(ParallaxComponent.class).get());
    }

    @Override
    protected void processEntity(Entity entity, float deltaTime) {

    }
}

File: core/src/bara/game/systems/ParallaxSystem.kt
package bara.game.systems

import com.badlogic.ashley.core.Entity
import com.badlogic.ashley.core.Family
import com.badlogic.ashley.systems.IteratingSystem
import bara.game.components.ParallaxComponent

class ParallaxSystem :
    IteratingSystem(Family.all(ParallaxComponent::class.java).get())
{
    //TODO: implement
    override fun processEntity(entity: Entity, deltaTime: Float) {

    }
}
